src/main/java/ch/njol/skript/expressions/ExprWorld.java

```diff
@@ -48,12 +49,13 @@
 @Description("The world the event occurred in.")
 @Examples({"world is \"world_nether\"",
 		"teleport the player to the world's spawn",
-		"set the weather in the player's world to rain"})
+		"set the weather in the player's world to rain",
+		"set {_world} to world of chunk at location of targeted entity"})
```

Comment:
Not a good example (`world of targeted entity`), you also don't have to add an example. If you do want to, do something like `set {_world} to world of {_chunk}` or `set {_world} to world of event-chunk`